samples/data-sealing/common/dispatcher.cpp

```diff
@@ -90,9 +90,9 @@ int ecall_dispatcher::seal_data(
     ret = cipher_data(
         ENCRYPT_OPERATION,
         m_data,
-        m_data_size,
+        (unsigned int)m_data_size,
```

Comment:
If we are going to cast instead of change the original type, and this is C++, let's prefer `static_cast` to C-casting.

Reply:
We should prefer to change the original types to be correct.

Reply:
+1 to changing types to not require casting